docs/src/static-pages/accessibility.md

```diff
@@ -243,22 +280,47 @@ Using "headings" allows screen reader users to quickly understand the content of
 
 ### **How**
 
-Use Headings #H1 to #H4 to structure a page to help screen reader users understand section hierarchy. 
-
-### **Do / Don't**
+Use Headings #H1 to #H6 to structure a page to help screen reader users understand section hierarchy. 
 
-- Do use Heading tags on all headings
 
 ### **Tips**
 - Use an appropriate Heading element instead of simply styling a paragraph to look like a heading
 - Heading copy should clearly describe the content underneath it
 - Headings can be hidden if not beneficial to sighted users
+- Don't skip heading levels
 - Note that the screen reader will read 'heading level 1' for example, after reading the actual heading
 
 ### **Tests**
 
 Use a screen reader and select "headings" mode from the rotor. Check that all headings are found.
 
+## Lists
+
+### **Why**
+
+When screen readers come to a properly marked up list, they will notify the user that there is a list and how many items are in it.
+
+### **Who**
+
+- Screen reader users
+- Robots (eg search engines)
+
+### **How**
+
+Mark the list as a real list with correct HTML mark up, rather than creating a fake list (with only visual styling) using elements such as `<br>` or `<p>`.
```

Comment:
```suggestion
Mark the list as a real list with correct HTML markup, rather than creating a fake list (with only visual styling) using elements such as `<br>` or `<p>`.
```